### PR TITLE
Fix links to Leo's paper

### PR DESCRIPTION
**Pull request by liyishuai**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -89,7 +90,7 @@ script:
   echo -e 'travis_fold:end:QuickChick.test.python-io'
 - echo 'Removing QuickChick...';
   echo -e 'travis_fold:start:QuickChick.remove';
-  opam remove coq-quickchick;
+  opam remove .;
```
Comment (Lysxia) [inline .travis.yml]: This looks asymmetrical. Why not
```
opam install coq-quickchick
...
opam remove coq-quickchick
```

File: README.md [hunk 2]
```diff
@@ -100,7 +100,7 @@ The main documentation is the DeepSpec summer school tutorial:
 Pretty soon this will become a software foundations volume!
 
 Here is some more reading material:
-  - Our POPL 2018 paper on [Generating Good Generators for Inductive Relations](http://www.cis.upenn.edu/~llamp/pdf/GeneratingGoodGenerators.pdf)
+  - Our POPL 2018 paper on [Generating Good Generators for Inductive Relations][mechanism for automatically deriving generators for inductive relations]
```
Comment (Lysxia) [inline README.md]: Uh, that's not a link.

Comment (liyishuai) [reply README.md]: It is, in reference style.

Comment (Lysxia) [reply README.md]: OH

**Pull request merged**